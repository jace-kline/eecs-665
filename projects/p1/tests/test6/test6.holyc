"Test\a123" 123