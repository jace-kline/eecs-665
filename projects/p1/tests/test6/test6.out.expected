INTLIT:123 [1,13]
EOF [1,16]